### DISCUSSION TITLE
Version 5.0.0.2

### DISCUSSION THREAD

**Discussion by rsjaffe** [rsjaffe/MIDI2LR#517] [Announcements]
_Build 2 fixes a potential macOS Monterey installer issue in build 1 of version 5. Users who successfully installed version 5.0.0.1 do not need to update._
## New features
  * Masking commands for Lightroom Classic Version 11. **Note: Versions 5 and later do not work properly with Lightroom Classic Versions 10 or earlier, as the changes Adobe made are not compatible with earlier versions.** Details are on [Commands Page](https://github.com/rsjaffe/MIDI2LR/wiki/Commands) under *Masking*.
  * **Profiles**: removal of *Brush Adjustments*, *Graduated Filter* and *Radial Filter* profiles, as those tools no longer exist as separate items in Lightroom Version 11. *Masking* profile added.
## Improvements
  * **Keywords**: can now assign 64 different keywords to commands (up from 16).
  * **Local presets**: can now assign 50 different local presets to commands (up from 16).
## Bug fixes
  * **macOS Russian and Thai keyboard users** can now use a-z (Latin alphabet) letters in keyboard shortcuts.
  * **Local presets** stopped functioning after upgrading to Lightroom version 11, fixed in this release.
## Removed commands
  * The following are no longer available as they are no longer relevant with the new masking features in Lightroom 11:
    * GraduatedFilter, RadialFilter, AdjustmentBrush, ShowMaskOverlay, EnableCircularGradientBasedCorrections, EnableGradientBasedCorrections, EnablePaintBasedCorrections, ResetCircGrad, ResetGradient, ResetBrushing
  * See the commands in *Masking* at the [Commands Page](https://github.com/rsjaffe/MIDI2LR/wiki/Commands) for their replacements.

<hr /><em>This discussion was created from the release <a href='https://github.com/rsjaffe/MIDI2LR/releases/tag/v5.0.0.2'>Version 5.0.0.2</a>.</em>

**Comment by Enricopez** [comment]
Still some issue with shortcut on keyboard. 
ex:
- "\" before and after doesn't work if you assign a shortcut on midi
- "H" shortcut does't work (show edit pins)
- toggle pick/unpick doesn't work
- Show spot removal doesn't work

All the Masking menù inside Midi2lr works great, but this new release has disabled the possibility to use and assign shortcut commands to the midi trough midi2lr

**Reply by rsjaffe** [reply]
The latest release doesn't change the keyboard functionality. If you are on a Windows machine, make sure Lightroom is the foreground application when checking keyboard shortcuts.

**Comment by Enricopez** [comment]
Thanks for your reply 🙏🏻
I absolutely sure on what you say, but doesn’t work. 
I have tried in every way, but don’t.
I'm on Mac (Mac mini with Monterey)

Trust me 🙏🏻

**Comment by eugeniobrito** [comment]
Hello!

First, i would like to thank you for your great work in Midi2LR.

I must say that my Show spot removal doesn't work also...

**Reply by rsjaffe** [reply]
That's odd. Spot removal hasn't changed in the latest release, either in MIDI2LR or Lightroom itself.

**Reply by eugeniobrito** [reply]
I think it is odd, also... 

But i checked this: if i select the Show spot removal with my mouse, it keeps selected (as it should be). And then, if i use the Midi2LR command "Show spot removal", it works, but it only deselects the tool. 
If i click again to activate tool with midi2lr command, it dosn't work.

Conclusion: it works if the tool was already enabled (and it disable it), but it doesn't work to enable it.

Sorry for my poor english... hope you understood...

**Reply by rsjaffe** [reply]
Fixed in 5.1
